src/config.ts

```diff
@@ -133,6 +140,11 @@ const buildAppConfig = async (secrets: any): Promise<AppConfig> => {
           dlq: process.env.KAFKA_REINDEXING_DLQ,
         },
       },
+      producers: {
+        publicRelease: {
+          topic: process.env.KAFKA_PUBLIC_RELEASE_TOPIC || 'public_release',
```

Comment:
lets be even more descriptive in our topic names to prevent confusion down the road... how about a default like `files_public_release`. We've been using the files_ as a kind of "pseudo-namespacing" for this service.